src/modules/file/services/file.service.ts

```diff
@@ -475,7 +469,7 @@ export class FileService {
         );
 
         this.logger.log(`creating iam token for file variant: ${uuid}`);
-        const iamService = await this.getCloudIAMService(
+        const iamService = this.getCloudIAMService(
```

Comment:
Could you please verify if this process is async or not? It should be async if there is any API call with google IAM service.

Reply:
not required